src/Stryker.Core/Stryker.Core/StrykerRunResult.cs

```diff
@@ -0,0 +1,26 @@
+using Stryker.Core.Initialisation.ProjectComponent;
+using Stryker.Core.Options;
+
+namespace Stryker.Core {
+    public interface IStrykerRunResult
```

Comment:
Why did you create an interface for this class? I don't think this has to be mocked somewhere.

Reply:
It has to be mocked in the `StrykerCLITest`, I don't want it to be dependend on the implementation of the object but rather the results.